### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by DennisDawson**
Came across a couple of minor typos.

### CODE REVIEW

File: content/use-cases/payments/restricting-deposits-uc.md [hunk 1]
```diff
@@ -7,22 +7,21 @@ labels:
...
+To comply with banking regulations, financial institutions must provide documentation about the sources of funds they receive. These regulations seek to prevent illicit activity by requiring institutions to track the source and destination of all payments they process. On the XRP Ledger, payments can be sent and received without any interaction from the receiver. This default behavior can be problematic, but you can enable deposit authorization so you only receive funds you explicitly approve.
 
 Accounts with deposit authorization enabled can only receive funds through:
 
   - Preauthorized Accounts
-  - Escrow
-  - Payment Channels
   - Checks
+  - Escrow
+<!-- - Payment Channels -->
```
Comment (oeggert) [inline content/use-cases/payments/restricting-deposits-uc.md:17]: Aren't payment channels also a valid way of receiving funds?

**Review by oeggert — APPROVED** [review]
Looks good to me--only question is about payment channels being commented out.

File: content/use-cases/payments/restricting-deposits-uc.md (hunk 1 above)
Comment (DennisDawson) [reply content/use-cases/payments/restricting-deposits-uc.md:17]: The section on payment channels is commented out. If we're not ready to discuss Payment Channels in this context, I think it's better to omit the bullet point.

Comment (oeggert) [reply content/use-cases/payments/restricting-deposits-uc.md:17]: Makes sense to me. No other questions!

**Pull request merged**